docs/changelog.rst

```diff
@@ -0,0 +1,16 @@
+.. _changelog:
+
+Changelog
+=========
+
+Version 0.2.0
```

Comment:
Should it be 0.3.0?

Reply:
I don't know, is it current version or future one ? It is hard to know because you can't really plan in advance what will be the number of the futur version (i.e. we don't know if we won't include some drastic changes in the next version which will change the first number). Maybe a more easy to understand would be 

```
Unreleased 
---------------

blah blah

Version 0.2.1
-----------------

blah blah
```

Reply:
I think it's a good idea. Instead of "unreleased" we could put "development" or "under development"

Reply:
Something like that ok ?